Formula/libzt.rb

```diff
@@ -0,0 +1,32 @@
+class Libzt < Formula
+  desc "Encrypted P2P networking library for applications (GPLv3)"
+  homepage "https://www.zerotier.com"
+
+  url "https://github.com/zerotier/libzt.git",
+    :tag      => "1.3.1-hb0",
```

Comment:
Can we expect the next version to be a non-`hb` tag? To be explicit: this will be fine for now as long as Homebrew can get back onto "normal" versions next.

Reply:
Yes, the next version will be a non-`hb` tag. For instance it's likely a `1.3.2` will be next.